Eatery Blue/UI/MenuPickerSheet/UpcomingMenuPickerSheetViewController.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  UpcomingMenuPickerSheetViewController.swift
+//  Eatery Blue
+//
+//  Created by Tiffany Pan on 9/29/23.
+//
+
+import EateryModel
+import UIKit
+
+protocol UpcomingMenuPickerSheetViewControllerDelegate: AnyObject {
+    
+    func upcomingMenuPickerSheetViewController(_ vc: UpcomingMenuPickerSheetViewController, didSelectMenuChoiceAt index: Int)
+    func upcomingMenuPickerSheetViewController(_ vc: UpcomingMenuPickerSheetViewController, didChangeMenuChoice string: String)
+    
+}
+
+class UpcomingMenuPickerSheetViewController: SheetViewController {
+    
+    weak var delegate: UpcomingMenuPickerSheetViewControllerDelegate?
+    
+    private var menuChoices = ["Breakfast", "Lunch", "Dinner", "Late Dinner"]
+    private var menuTimes = ["7:00 AM - 2:30 PM", "10:30 AM - 2:30 PM", "4:30 PM - 9:00 PM", "8:30 PM - 10:30 PM"]
+    
+    private var menuChoiceViews: [UpcomingMenuChoiceView] = []
+    var selectedMenuIndex: Int?
+    
+    func setUp() {
+        addHeader(title: "Menus")
+        addMenuChoiceViews()
+        addPillButton(title: "Show menu", style: .prominent) { [self] in
+            if let selectedIndex = self.selectedMenuIndex {
+                delegate?.upcomingMenuPickerSheetViewController(self, didSelectMenuChoiceAt: selectedIndex)
```

Comment:
Not sure if you managed to get to this but on call you mentioned that you may only need one of two (either the selectedIndex or the actual menu choice) but if so, you should remove the other one